### DISCUSSION TITLE
10/10 amazing

### DISCUSSION THREAD

**Discussion by Ticha22608** [hazeldritch/ParticleSFX#1] [General]
congratulations on your project i am sooo so proud of u i wish u the best

**Comment by hazeldritch** [comment]
![im going to yell](https://github.com/hmzel/ParticleSFX/assets/96555008/1384e2df-bc9b-4683-b3d3-475d20acb413)

**Reply by Ticha22608** [reply]
send more of these pls

**Reply by hazeldritch** [reply]
![leave me alone](https://github.com/hmzel/ParticleSFX/assets/96555008/005e2ceb-bda3-4358-bf63-a3a7650b60ad)

**Reply by hazeldritch** [reply]
![sunset](https://github.com/hmzel/ParticleSFX/assets/96555008/86f0c3ca-ec90-49c0-8906-f2e5c190111f)

**Reply by Arti-Dev** [reply]
![eevee-rasputin](https://github.com/hmzel/ParticleSFX/assets/46334223/55f15adf-1127-4d95-8bed-d69426da1f60)
